pkgs/by-name/sm/smtp4dev/package.nix

```
@@ -0,0 +1,55 @@
+{
+  buildDotnetModule,
+  fetchFromGitHub,
+  nodejs,
+  npmHooks,
+  gcc,
+  fetchNpmDeps,
+  dotnetCorePackages,
+  lib
+}:
+let
+  version = "3.3.4";
+  src = fetchFromGitHub {
+    owner = "rnwood";
+    repo = "smtp4dev";
+    rev = version;
+    hash = "sha256-ARq5OpFJ4o9KdBXvzOx7QLB8GNfmXWjO0RR4jKP8qRI=";
+  };
+  npmRoot = "Rnwood.Smtp4dev/ClientApp";
+in
+buildDotnetModule {
+  inherit version src npmRoot;
+  pname = "smtp4dev";
+
+  nativeBuildInputs = [
+    nodejs
+    nodejs.python
+    npmHooks.npmConfigHook
+    gcc
```

Seems to be working with `clang` and `stdenv.cc`
```suggestion
    stdenv.cc
```

---

If I understand that correctly, we need a C compiler for npm dependencies? A comment explaining that would be nice to have.